docs/sphinx/cookbook/move_to_managed.rst

```diff
@@ -9,7 +9,9 @@ should be moved to unified memory in order to make it accessible by the GPU.
 You can do this with the :func:`umpire::ResourceManager::move` operation:
 
 .. literalinclude:: ../../../examples/cookbook/recipe_move_to_managed.cpp
-                    :lines: 26
+   :start-after: _umpire_tut_move_host_to_managed_start
```

Comment:
This is another spot where I think too much code is included in the snippet. What do you think about only including the code below the "// Move data to unified memory" comment?